extensions/drupal-org/src/search-change-records.tsx

```diff
@@ -1,7 +1,7 @@
 import { Action, ActionPanel, Icon, List } from "@raycast/api";
 import { useEffect, useState } from "react";
-import { SearchChangeRecordsState } from "./types";
-import { getDrupalChangeRecords } from "./utils/do-change-record-api";
+import { SearchChangeRecordsState } from "./utils/search-change-records.ts/types";
+import { getDrupalChangeRecords } from "./utils/search-change-records.ts/do-change-record-api";
```

Comment:
Any reason there is a `.ts` in the name of the directory?

Reply:
No, that's a mistake on my part. Thanks for noticing!